src/views/lighttable.c

```diff
@@ -2501,29 +2503,51 @@ void gui_init(dt_view_t *self)
   g_signal_connect(G_OBJECT(display_profile), "value-changed", G_CALLBACK(display_profile_callback), NULL);
 }
 
+static gboolean _is_custom_image_order_required(dt_view_t *self)
+{
+  gboolean custom_image_order_required = FALSE;
```

Comment:
can be removed with proposed change below

Reply:
Done

Reply:
Perfect! Thanks.